.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+
+---
+
+Your issue may already be reported!
+Please search on the [issue tracker](https://github.com/pytorch/serve/issues) before creating one.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+* torchserve version:
+* torch version:
+* torchvision version [if any]:
+* torchtext version [if any]:
+* torchaudio version [if any]:
+* java version:
+* Operating System and version:
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+...
+
+## Failure Logs [if any]
+<!--- Provide any relevant log snippets or files here. -->
+
+## Your Environment
```

Comment:
Can we move this section after context? This is important information and gets lost if there are a lot of steps to repro particular issue.

Reply:
Made required changes.